bpmn-model-api/generate-process-fluent-api/pom.xml

```diff
@@ -11,8 +11,7 @@
     <version.junit>4.13.2</version.junit>
     <version.h2>2.1.214</version.h2>
     <version.slf4j>1.7.26</version.slf4j>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.release>11</maven.compiler.release>
```

Comment:
❌ I just tried to run the example with this change but it's missing the bumped `maven-compiler-plugin` version to be able to use this. I assume this will be the case for most of the other examples as well.